src/core/lib/experiments/experiments.yaml

```diff
@@ -112,7 +112,7 @@
 - name: event_engine_dns
   description:
     If set, use EventEngine DNSResolver for client channel resolution
-  expiry: 2023/10/01
+  expiry: 2024/01/01
```

Comment:
please push this out a few weeks into January, so nobody gets nagged to act on an expiry over the winter holidays.

Reply:
Makes sense, done.